src/components/ArticlesList/ArticlesList.jsx

```diff
@@ -0,0 +1,54 @@
+// imports
+import { useEffect, useState } from "react";
+import exampleArticlesData from "../../SampleData/api-articles.json";
```

Comment:
I like the use of this but now that you have connected to your database this should be removed

Reply:
done